lib/core/domain/event/entities/event_ticket_types.dart

```diff
@@ -72,7 +73,8 @@ class PurchasableTicketType with _$PurchasableTicketType {
         id: dto.id,
         active: dto.active,
         addressRequired: dto.addressRequired,
-        cost: dto.cost,
+        priceCurrency: dto.cost?.keys.firstOrNull,
+        price: dto.cost?.values.firstOrNull?.cost,
```

Comment:
the dto already updated to use `prices` . Why `priceCurrency` and `price` here ?